blog/2024-01-25-inspecting-dependency-inference-results/index.mdx

```diff
@@ -0,0 +1,152 @@
+---
+    authors: [alexey]
+    tags: [concepts]
+---
+
+# Inspecting dependency inference results
+
+<CaptionedImg src={require("./splash.jpg").default}>
+  Photo by [Agence
+  Olloweb](https://unsplash.com/photos/magnifying-glass-near-gray-laptop-computer-d9ILr-dbEdg)
+  /
+  [Unsplash](https://unsplash.com/photos/magnifying-glass-near-gray-laptop-computer-d9ILr-dbEdg)
+</CaptionedImg>
+
+{/* truncate */}
+
+An important distinguishing feature of Pants v2 is its ability to automatically [infer your code's internal and external dependencies](https://www.pantsbuild.org/blog/2022/10/27/why-dependency-inference). The information about them is available via the `dependencies` and `peek` goals, so that metadata about what modules import from modules is always readily accessible.
```

Comment:
```suggestion
An important distinguishing feature of Pants v2 is its ability to automatically [infer your code's internal and external dependencies](https://www.pantsbuild.org/blog/2022/10/27/why-dependency-inference). The information about those inferences is available via the `dependencies` and `peek` goals, so that metadata about what modules import from other modules is always readily accessible to you.
```